app/models/concerns/foreman_ansible/hostgroup_extensions.rb

```diff
@@ -0,0 +1,20 @@
+module ForemanAnsible
+  # Relations to make Host::Managed 'have' ansible roles
+  module HostgroupExtensions
+    extend ActiveSupport::Concern
+
+    included do
+      has_many :hostgroup_ansible_roles, :foreign_key => :hostgroup_id
+      has_many :ansible_roles, :through => :hostgroup_ansible_roles,
+                               :dependent => :destroy
+
+      def all_ansible_roles
```

Comment:
If you end up doing  https://github.com/theforeman/foreman_ansible/pull/30/files#r74576083 , this method could also go in the concern